deploy/README.md

```diff
@@ -35,3 +35,16 @@ jobs:
           automerge: true
           upgrade-commands: '{"tapestry-react":"yarn tr upgrade"}' # this is assuming that you're upgrading
 ```
+
+## Config Files in Consumers
+
+Some consumers of the library will have specific commands that they need to use to upgrade to the newest version. Instead of making each library know that, you can add a config file to the consumer repo called `.pco-release.config.yml`.
+
+```yml
+# `./.pco-release.config.yml`
+upgrade_command: "yarn tr upgrade"
```

Comment:
Does this need to be configured per-package, like above: you can specify an upgrade command per npm package if I understand it correctly.

Reply:
If I understand what you're asking is that the current config requires you to specify the repo that the config applies to (ie. `{ "tapestry-react": "yarn tr upgrade" }`), but this new config does not require that scoping.

The reason for that is that this config file will live inside the tapestry-react repo so that all libraries that attempt to upgrade to `tapestry-react` will use the `yarn tr upgrade` command to upgrade, rather than each library needing to specify that it upgrades with that command.  Since it is defined inside the repo that specifies it, there is not a need for that scoping in the config anymore.

There is a scenario where there is a faint possibility that a configuration will need to specify a separate config for how that library specifically updates (I can't see a real world reason we would need this now), but in that case, I think the responsibility would be to specify that setting within the action and we would add the setting.

Reply:
My bad, I just misunderstood!